.gitignore

```diff
@@ -23,6 +23,7 @@ Custom_scripts/
 Data_Files/MicroscopyCalibration/Files/
 .pytest_cache/
 .python-version
+wranglertools/.config/gspread/authorized_user.json
```

Comment:
If there is nothing worth tracking in `.config` I would gitignore the whole thing just to be safe... You also may want to provide another means to specify these credentials since folks who install from PyPi won't have this path.